src/main/internals/lodash.ts

```diff
@@ -0,0 +1,70 @@
+// export function isEmpty(obj: unknown): boolean {
+//     if (obj === undefined || obj === null) {
+//         return true
+//     }
+
+//   if (typeof obj === 'object') {
+//     return [Object, Array].includes(obj.constructor ) && !Object.entries(obj || {}).length
+//   }
+// }
+
+/** Used to check objects for own properties. */
+const hasOwnProperty = Object.prototype.hasOwnProperty
+
+/**
+ * Checks if `value` is an empty object, collection, map, or set.
+ *
+ * Objects are considered empty if they have no own enumerable string keyed
+ * properties.
+ *
+ * Array-like values such as `arguments` objects, arrays, buffers, strings, or
+ * jQuery-like collections are considered empty if they have a `length` of `0`.
+ * Similarly, maps and sets are considered empty if they have a `size` of `0`.
+ *
+ * @since 0.1.0
+ * @category Lang
+ * @param {*} value The value to check.
+ * @returns {boolean} Returns `true` if `value` is empty, else `false`.
+ * @example
+ *
+ * isEmpty(null)
+ * // => true
+ *
+ * isEmpty(true)
+ * // => true
+ *
+ * isEmpty(1)
+ * // => true
+ *
+ * isEmpty([1, 2, 3])
+ * // => false
+ *
+ * isEmpty('abc')
+ * // => false
+ *
+ * isEmpty({ 'a': 1 })
+ * // => false
+ */
+function isEmpty(value: unknown): boolean {
```

Comment:
I'm intending to include vendored libnraries that are used not much throughout the SDK, just for turning down the final dependency size of the SDK itself.

For example if as an SDK, you're required to install Lodash that'll take a few KBs, we can trim that down into a few bytes not with using tree shaking, but with vendoring the needed functions.